src/utils/helper-functions.tsx

```diff
@@ -114,3 +114,31 @@ export const spliceSx = (defaultStyles?: SxProps, newStyles?: SxProps) =>
 
 export const studentOrTutor = (userRole: '' | UserRole) =>
   userRole === UserRoleEnum.Tutor ? UserRoleEnum.Tutor : UserRoleEnum.Student
+
+export const getTimeAtS2S = (creationDate: string, currentDate: string) => {
+  const difference: number = new Date(currentDate) - new Date(creationDate)
+
+  const conversionToDays: number = Math.round(difference / (1000 * 3600 * 24))
+
+  if (conversionToDays >= 365) {
+    const years = Math.floor(conversionToDays / 365)
+    return { number: years, format: years === 1 ? 'Year' : 'Years' }
+  }
+
+  if (conversionToDays >= 30 && conversionToDays < 365) {
```

Comment:
you dont need second part of statement, because if years > 1 this if statement wont work because of return